src/components/BigNumberPad.js

```diff
@@ -18,12 +18,16 @@ const propTypes = {
     /** Used to locate this view from native classes. */
     nativeID: PropTypes.string,
 
+    /** Whether long press is disabled */
+    isDisabledLongPress: PropTypes.bool,
```

Comment:
I think I'd prefer `isLongPressDisabled` but not a big deal :smile:

Reply:
👍🏻